main_test.go

```diff
@@ -361,6 +362,43 @@ func TestAlises(t *testing.T) {
 	}
 }
 
+func TestParseIniFile(t *testing.T) {
+	configHomeEnv := "XDG_CONFIG_HOME"
+	originalConfigHome := os.Getenv(configHomeEnv)
+	defer os.Setenv(configHomeEnv, originalConfigHome)
+
+	tests := []struct {
+		message       string
+		configPath    string
+		expectedError string
+	}{
+		{
+			message:    "valid ini file should parse correctly",
+			configPath: path.Join("testdata", "ini", "valid"),
+		},
+		{
+			message:    "absent ini file should be ignored",
+			configPath: path.Join("testdata", "ini", "missing"),
+		},
+		{
+			message:       "invalid ini file should raise error",
+			configPath:    path.Join("testdata", "ini", "invalid"),
+			expectedError: "couldn't read testdata/ini/invalid/yab/defaults.ini: testdata/ini/invalid/yab/defaults.ini:2: time: unknown unit foo in duration 3foo",
+		},
+	}
+	for _, tt := range tests {
+		os.Setenv(configHomeEnv, tt.configPath)
+		parser, _ := newParser()
+		err := parseDefaultConfigs(parser)
+		if tt.expectedError == "" {
+			assert.NoError(t, err, tt.message)
+		} else {
+			assert.Error(t, err, tt.message)
+			assert.Equal(t, err.Error(), tt.expectedError, tt.message)
```

Comment:
use `assert.EqualError` and put the expected error as the first arg. then you can drop the `assert.Error`, just `assert.EqualError(t, tt.expectedError, err, tt.message)`